src/components/common/Header.astro

```diff
@@ -13,17 +14,25 @@ const {
   image,
   headingColor,
   subheadingColor,
+  isContain,
```

Comment:
The isContain has issues with the navbar right? I also tried and this wont work. Can we instead change to removeHeader prop and layout.tsx not render that component only? Then we can use normal <img/> in the specific cohort pages

Reply:
the isContain is the new prop i added, it was not there previously. and the navbar is dynamically loading based on it has the "itContain" flag or not. it is happening from the layout.tsx actually. can you explain what issue you've noticed?

Reply:
Navbar should always be there. And thats hard to do with current approach. Am i right?

Reply:
i thought we need to remove the navbar in the cohort pages